jamal-snippet/src/main/java/javax0/jamal/snippet/Sort.java

```diff
@@ -0,0 +1,134 @@
+package javax0.jamal.snippet;
+
+import javax0.jamal.api.*;
+import javax0.jamal.tools.InputHandler;
+import javax0.jamal.tools.Params;
+import javax0.jamal.tools.Scan;
+
+import java.text.Collator;
+import java.util.*;
+import java.util.function.Function;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+import static javax0.jamal.tools.InputHandler.skipWhiteSpaces;
+import static javax0.jamal.tools.Params.holder;
+
+public class Sort implements Macro, InnerScopeDependent {
```

Comment:
Why is this macro `InnerScopeDependent`?

Implementing this interface is outdated. It was used to support config macros replaced by options now. If the option is to be defined locally then it has to use the option syntax and not define a macro inside the use of the `sort` macro.